activerecord/test/cases/type/float_test.rb

```diff
@@ -0,0 +1,12 @@
+require "cases/helper"
+
+module ActiveRecord
+  module Type
+    class FloatTest < ActiveRecord::TestCase
+      def test_float_cast_for_schema_with_infinity_value
+        type = Float.new
+        assert_equal '::Float::INFINITY', type.type_cast_for_schema(::Float::INFINITY)
```

Comment:
This is not the proper way to test this, please test that the schema dumper generates proper values.